website/docs/api/plugins/plugin-pwa.md

```diff
@@ -142,7 +142,7 @@ The [`standalone` strategy](https://petelepage.com/blog/2019/07/is-my-pwa-instal
 
 ### `injectManifestConfig` {#injectmanifestconfig}
 
-[Workbox options](https://developers.google.com/web/tools/workbox/reference-docs/latest/module-workbox-build#.injectManifest) to pass to `workbox.injectManifest()`. This gives you control over which assets will be precached, and be available offline.
+[Workbox options](https://developer.chrome.com/docs/workbox/reference/workbox-webpack-plugin/#type-InjectManifest) to pass to `workbox.injectManifest()`. This gives you control over which assets will be precached, and be available offline.
```

Comment:
It should be https://developer.chrome.com/docs/workbox/reference/workbox-build/#type-InjectManifestOptions or something similar. This option is passed to `workbox-build`, not `workbox-webpack-plugin`.